infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -125,6 +125,10 @@ spec:
                 echo "account = $JUPYTERHUB_USER" >> /opt/rucio/etc/rucio.cfg;
                 echo "auth_type = oidc" >> /opt/rucio/etc/rucio.cfg;
                 echo "oidc_audience = rucio" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_scope = openid profile offline_access" >> /opt/rucio/etc/rucio.cfg;
+                echo "oidc_issuer = escape" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_active true" >> /opt/rucio/etc/rucio.cfg;
+                echo "auth_oidc_refresh_before_exp 20" >> /opt/rucio/etc/rucio.cfg;
```

Comment:
I also have in my config the following line
`oidc_polling = true`

Reply:
I added it.